sites/partners/src/lib/listings/formTypes.ts

```diff
@@ -156,7 +156,7 @@ export const formDefaults: FormListing = {
   householdSizeMax: 0,
   householdSizeMin: 0,
   neighborhood: undefined,
-  region: RegionEnum.Greater_Downtown,
```

Comment:
chore: you can remove the regionEnum import now that it is no longer being used

Reply:
Adressed in: [aaddc7a](https://github.com/bloom-housing/bloom/pull/4824/commits/aaddc7a875f8832cd268aee8be507ea44ce98abe)